site/content/kapp-controller/_index.html

```diff
@@ -3,49 +3,65 @@
 ---
 
 <div class="hero subpage kapp-controller">
-    <div class="wrapper clearfix">
-        <div class="text-block">
-            <h1>Lightweight continuous delivery to Kubernetes</h1>
-            <p>Codify your application deployment in a first-class Kubernetes resource &mdash; App CR. Have Kubernetes fetch application configuration from various sources, template and customize it, and see it converge continuously.</p>
-            <div class="buttons">
-                <a class="button" href="#examples">See Examples</a>
-                <a class="button tertiary" href="https://github.com/vmware-tanzu/carvel-kapp-controller/releases" target="_blank">Install</a>
-            </div>
-        </div>
+  <div class="wrapper clearfix">
+    <div class="text-block">
+      <h1>Continuous delivery and package management for Kubernetes</h1>
+      <p>kapp-controller's declarative APIs and layered approach enables you to build, deploy, and manage your own 
+        applications. It also helps package your software into easily distributable packages and enables your users 
+        to discover, configure, and install these packages on a Kubernetes cluster.
+      </p>
+      <div class="buttons">
+        <a class="button" href="#examples">See Examples</a>
+        <a class="button tertiary" href="https://carvel.dev/kapp-controller/docs/latest/install/"
+          target="_blank">Install</a>
+      </div>
     </div>
+  </div>
 </div>
 
 <div class="wrapper mobile-blue">
-    <div class="yaml-grid grid three">
-        <div class="col">
-            <div class="icon">
-                <img src="/img/structure-oriented.svg" />
-            </div>
-            <h3>Consistent</h3>
-            <p>App CR provides a common consistent API for configuring and deploying your applications regardless if it is a Helm chart, plain YAML configuration, ytt templates, etc.</p>
-        </div>
-        <div class="col">
-            <div class="icon">
-                <img src="/img/modularized-config.svg" />
-            </div>
-            <h3>Composable</h3>
-            <p>Take advantage of different tools for each stage -- fetch, template and deploy. For example, fetch Helm templates from a Git repository, use Helm to template them, make modifications with ytt overlays, and finally deploy with kapp.</p>
-        </div>
-        <div class="col">
-            <div class="icon">
-                <img src="/img/deterministic.svg" />
-            </div>
-            <h3>Lightweight</h3>
-            <p>kapp-controller delegates all the heavy lifting to existing configuration management and deployment tools. In doing so, kapp-controller itself is kept very simple, and therefore provides a high degree of reliability.</p>
-        </div>
+  <div class="yaml-grid grid three">
+    <div class="col">
+      <div class="icon">
+        <img src="/img/structure-oriented.svg" />
+      </div>
+      <h3>App Developers</h3>
```

Comment:
"App Developers" might not be a good phrase to use.

Reply:
Do you have anything to suggest? Kubernetes Developers? It's a bit unclear to me what users we want to reach with the App CR.

Reply:
Changed to k8s developers for now. Let's iterate on this and come back to it.